src/index.tsx

```diff
@@ -1,14 +1,29 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import { HashRouter, Route, Routes } from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { HomePage } from './components/HomePage/HomePage';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
 
 createRoot(document.getElementById('root') as HTMLDivElement)
   .render(
-    <Router>
-      <App />
-    </Router>,
+    <HashRouter>
+      <Routes>
+        <Route path="/:path?" element={<App />}>
+          <Route index element={<HomePage />} />
+          <Route path="people" element={<PeoplePage />}>
```

Comment:
You can get rid of `element={<PeoplePage />` in this line, because later you say `with or without :selectedSlug render <PeoplePage />`